scripts/pre_commit_linter.py

```diff
@@ -39,6 +39,9 @@
 2.  To lint all files in  the folder or to lint just a specific file
         python scripts/pre_commit_linter.py --path filepath
 
+3.  To lint a specific list of files (*.js/*.py only)
+        python scripts/pre_commit_linter.py --files file1, file2 ... filen
```

Comment:
Are users supposed to have spaces between the filenames? I'd also call them file_1, file_2, etc. -- it's a bit hard to parse "filen" at the end.